pkg/api/v1/register.go

```diff
@@ -23,8 +23,11 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
+// GroupName is the group name use in this package
+const GroupName = api.GroupName
```

Comment:
Yes-- if we had an import tree test, this would have broken it :)
pkg/apis/x/v1 may not import pkg/apis/x.

On Tue, Dec 8, 2015 at 11:07 AM, Chao Xu <notifications@github.com> wrote:

> In pkg/api/v1/register.go
> <https://github.com/kubernetes/kubernetes/pull/18353#discussion_r46997962>
> :
>
> > @@ -23,8 +23,11 @@ import (
> >  	"k8s.io/kubernetes/pkg/runtime"
> >  )
> >
> > +// GroupName is the group name use in this package
> > +const GroupName = api.GroupName
>
> I think what @lavalamp <https://github.com/lavalamp> means is const
> GroupName = "" here, so that user can import v1 without importing the
> internal API (after we remove conversions).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/18353/files#r46997962>.
>